bin/grocery-delivery

```diff
@@ -62,7 +62,7 @@ end
 
 def read_checkpoint
   GroceryDelivery::Log.debug("Reading #{checkpoint_path}")
-  File.exists?(checkpoint_path) ? File.read(checkpoint_path).strip : nil
+  File.exist?(checkpoint_path) ? File.read(checkpoint_path).strip : nil
```

Comment:
I'm fine wit hthis, but we disabled that rule which makes me think you're not using the config.